File: tugas/python/Ram.py
class Ram():
    # constructor
    def __init__(self, capacity=0, price=0):
        self.__capacity = capacity
        self.__price = price

    # setter n getter
    def setCapacity(self, capacity):
        self.__capacity = capacity

    def getCapacity(self):
        return self.__capacity
        
    def setPrice(self, price):
        self.__price = price

    def getPrice(self):
        return self.__price

File: tugas/cpp/Ram.cpp
#include <bits/stdc++.h>

using namespace std;

class Ram{
    private:
        // atribut
        int capacity, price;

    public:
        // constructor
        Ram(){
            this->capacity = 0;
            this->price = 0;
        }

        Ram(int capacity, int price){
            this->capacity = capacity;
            this->price = price;
        }

        // setter n getter
        void setCapacity(int capacity){
            this->capacity = capacity;
        }

        int getCapacity(){
            return this->capacity;
        }

        void setPrice(int price){
            this->price = price;
        }

        int getPrice(){
            return this->price;
        }

        ~Ram(){};
};